ml-data-hub-plugin/src/test/resources/job-test/batch2.json

```diff
@@ -0,0 +1,14 @@
+{
+  "batch": {
+    "jobId": "10584668255644629399",
+    "batchId": "11345653515268525918",
+    "processor": "Mapping",
+    "stepNumber": "1",
+    "batchStatus": "failed",
+    "timeStarted": "2019-02-05T21:39:37.284127-08:00",
+    "timeEnded": "N/A",
```

Comment:
Just a note - we really shouldn't be putting "N/A" here, as there's a range index on this field. If we don't have a value, the right thing to do is not define the field. It's something we can clean up later, but in the future, I think it's better to not keep adding "N/A" in additional places.